### DISCUSSION TITLE
Trying to Change Page Title

### DISCUSSION THREAD

**Discussion by evanwilliammiller** [Lissy93/dashy#1156] [Help and Support]
Please help.  I've been at this for 8 hours.  Very frustrated.

All I want to do is change the word Dashy in the broswer tab.  I have crawled through every file in the /app/ folder in the docker instance and I have changed Dashy out in every single .json and .html I can find. I have changed it in the GUI.  I've changed it absolutely everywhere I can find.

I am no longer using my cloudflare tunnel.  All of this work is now happening via local IP so I know that's not the issue.

Someone please tell me how to change this config setting.  I have my name popping up during initialization but then when the page loads it goes back to Dashy.  I've cleared the browser cache more times than I can count now.

I'd be eternally grateful for any help in changing this setting.  I'm not referring to the page heading.  I'm talking about the TITLE that shows up in the tabs or in the tabs view in iOS.

Thank you in advance for any help.

**Comment by evanwilliammiller** [comment]
![dashy-title](https://user-images.githubusercontent.com/118140493/229320042-b4092603-d696-40ca-b01b-9816ee828698.png)

**Comment by talopine** [comment]
I have the same question.

**Comment by rdarshan927** [comment]
did anyone find it out!!! ... if yes, please write here !!!

**Comment by songyp0505** [comment]
same question too!

**Comment by gdeans** [comment]
As far as I can tell, the title item is the title of the page, not the title of the app as seen by the browser.  I cannot see any config item to define that so I would assume you'd have to crawl through the source code to find it, then modify it on your install.  However, the next time you update it'll be overwritten.  Maybe there's a feature request (idea) for this?

**Reply by gdeans** [reply]
Supposedly fixed in [#663](https://github.com/Lissy93/dashy/pull/663) but it's not working on my new install either.

**Comment by mkhinda29** [comment]
I was able to change the tab title, but the code had to be built/compiled on startup using my configuration file to pick up the change.  The added startup time isn't ideal though.
If you are using dashy in a docker container like I am, you can set the startup command to `command: ["yarn",  "build-and-start"]` and it will pick up your title setting.
https://github.com/Lissy93/dashy/issues/645#issuecomment-1864993038
